### DISCUSSION TITLE
[@types/node]The highwatermark option is missing from WriteStreamOptions

### DISCUSSION THREAD

**Discussion by Rollphes** [DefinitelyTyped/DefinitelyTyped#66685] [Issues with a @types package] [answered]
I was originally using 20.4.9, but this occurred when I updated to 20.6.0.
I found out that the cause was probably that it had disappeared from Node Docs.
I would like to have this fixed.
Also, this is the first time I have submitted an Issue to a large project, so if there are any flaws, please point them out.

https://github.com/nodejs/node/issues/49420
https://github.com/nodejs/node/pull/49456

[a comment by an automated account is not shown]

**Comment by Semigradsky** [comment, marked as answer]
I am on it

**Reply by Semigradsky** [reply]
@Rollphes Is it only for v20, or for v18/v16/v14 too? Do you know?

**Reply by Rollphes** [reply]
@Semigradsky Perhaps, but I think it's just v20.
I've been using it for a while and this is the first time it's disappeared.

**Reply by Semigradsky** [reply]
Ok, I checked the tests and see that it's applicable to all supported versions.

For example: https://github.com/nodejs/node/blob/e9e68b942d2242de7f8ef7697cf5f62dd5f325d6/test/parallel/test-file-write-stream2.js#L66-L68

**Reply by Semigradsky** [reply]
Fix here: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/66692

**Reply by Rollphes** [reply]
Thank you!!
